package.json

```diff
@@ -99,7 +99,7 @@
       "babel-plugin-transform-es2015-modules-commonjs",
       "transform-object-rest-spread",
       "transform-async-to-generator",
-      "babel-jest-assertions",
+      "./node_modules/babel-jest-assertions",
```

Comment:
```suggestion
      "module:babel-jest-assertions",
```

?

Reply:
I thought that was a Babel 7 syntax? If I'm mistaken, then yea, I prefer that (I did it in the Babel 7 PR already)